cpp/README.md

```diff
@@ -320,7 +320,7 @@ You can also fix any formatting errors automatically:
 
     make format
 
-These commands require `clang-format-5.0` (and not any other version).
+These commands require `clang-format-6.0` (and not any other version).
```

Comment:
Could you also update the `brew install llvm@5` command to `llvm@6` a couple lines below? Just tested it on macOS and it works fine.

Reply:
Done